### PR TITLE
Update operator version in rancher using updatecli

### PR DESCRIPTION
**Pull request by alexander-demicev**
This PR adds a workflow to update the operator version in rancher using updatecli.

### CODE REVIEW

File: .updatecli/updatecli.d/bump-rancher-dep.yaml [hunk 1]
```diff
@@ -0,0 +1,137 @@
+name: 'Bump EKS operator'
+
+scms:
+  rancher:
+    kind: github
+    spec:
+      user: highlander-ci-bot
```
Comment (furkatgofurov7) [inline .updatecli/updatecli.d/bump-rancher-dep.yaml:7]: should we align all rancher bump flows using the same bot in all operators? currently it is only in [aks](https://github.com/rancher/aks-operator/blob/master/.updatecli/updatecli.d/bump-rancher-dep.yaml#L7) operator but could be in gke operator in the future

File: .updatecli/updatecli.d/bump-rancher-dep.yaml [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        sourceid: rancher-gosum
+    rancher-apis-gomod:
+        name: Update EKS operator version to "{{ requiredEnv "UPDATECLI_EKS_OPERATOR_TAG" }}" in pkg/apis/go.mod
+        kind: file
+        spec:
+          file: pkg/apis/go.mod
+        scmid: rancher
+        sourceid: rancher-apis-gomod
+    rancher-apis-gosum:
+        name: Update EKS operator version to "{{ requiredEnv "UPDATECLI_EKS_OPERATOR_TAG" }}" in pkg/apis/go.mod
```
[the remark this answers is not in the document]
Comment (furkatgofurov7) [reply .updatecli/updatecli.d/bump-rancher-dep.yaml:122]: I noticed it has to be fixed in https://github.com/rancher/aks-operator/blob/master/.updatecli/updatecli.d/bump-rancher-dep.yaml#L122 too

**Review by furkatgofurov7 — COMMENTED** [review]
Great, left one question and one nit

File: .updatecli/updatecli.d/bump-rancher-dep.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexander-demicev) [reply .updatecli/updatecli.d/bump-rancher-dep.yaml:122]: thanks!

File: .updatecli/updatecli.d/bump-rancher-dep.yaml (hunk 1 above)
Comment (alexander-demicev) [reply .updatecli/updatecli.d/bump-rancher-dep.yaml:7]: Yes, the bot didn't exist when aks PR was created. I will update the config there too.

**Comment by alexander-demicev** [comment]
replaced by https://github.com/rancher/eks-operator/pull/145

**Pull request closed without merging**